view/index.html

```diff
@@ -13,14 +13,15 @@
     <script src="./js/services/d3.v4.js"></script>
     <link rel="stylesheet" href="./css/main.css">
 </head>
-<body>
-    <header data-ng-include="'components/header.html'"></header>
+<body background="./view/assest/Background/MainBG.png">
+    <header data-ng-include="'components/header.html'" ></header>
 
     <!-- sidebar -->
     <div data-ng-include="'components/sidebar.html'"></div>
-
+    
```

Comment:
remove trailing spaces from every file when u fix the above errors. also, add an empty line in the last of the fine.